### DISCUSSION TITLE
Curious about Rust usage in this project

### DISCUSSION THREAD

**Discussion by dandv** [toeverything/AFFiNE#9065] [General]
Found this repo as the [3rd most starred Rust project on GitHub](https://github.com/topics/rust-lang).

I'm curious what portion of AFFiNE (not its dependencies) is written in Rust.

**Comment by Brooooooklyn** [comment]
Backend: https://github.com/toeverything/AFFiNE/tree/canary/packages/backend/native
Client: https://github.com/toeverything/AFFiNE/tree/canary/packages/frontend/native

Recently, our storage layer (Electron/iOS/Android) is also being implemented in Rust, and the proportion of Rust code is expected to increase significantly.
